### PR TITLE
[Reposition point] Update UI for confirming/cancelling move operations

### PR DESCRIPTION
**Pull request by DaoyuT**
Also fixed a bug where the close button in feature panel does not deselect the selected marker.

### CODE REVIEW

**Comment by gino-m** [comment]
@DaoyuT We spoke, since this is stilL WIP can you please ping when ready for another look?

**Comment by DaoyuT** [comment]
> @DaoyuT We spoke, since this is still WIP can you please ping when ready for another look?

Added a change to subscribe to selected feature/observation change and cancel the reposition.

File: web-ng/src/app/components/map/map.component.css [hunk 1]
```diff
@@ -13,4 +13,48 @@
...
+
+.reposition-confirm-dialog {
+  position: absolute;
+  bottom: 2em;
+  left: 50%;
+  transform: translate(-50%, 0);
+  width: 407px;
+  height: 66px;
+  border-radius: 8px;
+  background: #188038;
```
Comment (gino-m) [inline web-ng/src/app/components/map/map.component.css:25]: Can we use style vars and theme for colors in this sheet? See other scss for examples

File: web-ng/src/app/components/map/map.component.ts [hunk 2]
```diff
@@ -207,16 +229,34 @@ export class MapComponent implements AfterViewInit, OnDestroy {
     };
     const marker = new google.maps.Marker(options);
     marker.addListener('click', () => {
-      this.selectMarker(marker);
+      if (this.disableMapClicks) {
+        return;
+      }
       this.navigationService.selectFeature(feature.id);
     });
+    marker.addListener('dragstart', (event: google.maps.MouseEvent) => {
```
Comment (gino-m) [inline web-ng/src/app/components/map/map.component.ts:237]: Can we break these listeners out into their own functions for readability?

File: web-ng/src/app/components/map/map.component.css (hunk 1 above)
Comment (DaoyuT) [reply web-ng/src/app/components/map/map.component.css]: It seems this color is specific to this confirm dialog, extracted as a variable in the scss file.

File: web-ng/src/app/components/map/map.component.ts (hunk 2 above)
Comment (DaoyuT) [reply web-ng/src/app/components/map/map.component.ts]: Done.

**Review by gino-m — APPROVED** [review]
LGTM!

**Pull request merged**